users/admin.py

```diff
@@ -1,5 +1,5 @@
 from django.contrib import admin
-from .models import User, Apartment, City, Hobby, Seeker
+from .models import User, Hobby, Seeker
```

Comment:
I'm pretty sure that you can register models from multiple apps into a single admin. That way you have one place to go instead of an admin for each app.
This makes sense when the apps are all part of a single system.

Reply:
Ok, this really sounds better than the current state.
In which directory do you recommend putting it?